### PR TITLE
fix: make celestia-rpc to compile in wasm32 target

### PR DESCRIPTION
**Pull request by oblique**

### CODE REVIEW

**Review by zvolin — COMMENTED** [review]
maybe we could include the wasm build on CI?

**Comment by oblique** [comment]
Added a `build-wasm` job on CI, however I don't build tests. We need to implement more stuff for that.

File: celestia/Cargo.toml [hunk 1]
```diff
@@ -4,11 +4,15 @@ version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
 
+#[lib]
+#crate-type = ["cdylib"]
+
```
Comment (fl0rek) [inline celestia/Cargo.toml:9]: Are we planning on changing that for example for wasm? Or is it just leftover from editing?

Comment (oblique) [reply celestia/Cargo.toml]: fixed

**Review by fl0rek — APPROVED** [review]
:guitar:

**Review by zvolin — APPROVED** [review]

**Pull request merged**